packages/fxa-auth-server/lib/senders/emails/templates/cadReminderFirst/en.ftl

```diff
@@ -4,4 +4,4 @@ cadReminderFirst-action-plaintext = { cadReminderFirst-action }:
 # In the title of the email, "It takes two to sync", "two" refers to syncing two devices
 cadReminderFirst-title-1 = It takes two to sync
 cadReminderFirst-description-1 = Take your tabs across all your devices. Get your bookmarks, passwords, and other data everywhere you use { -brand-firefox }. It’s like having magic in your { -brand-firefox } account!
-cadReminderFirst-description-2 = It only takes a sec to sync.
```

Comment:
Verified this change with UX - this sentence was not quite accurate, and removing it simplifies the copy and removes the need for adding `part1` to the new version of previous ftl string. 
Kept the v2 though to make sure this will be flagged as a new string for localization.